backend/routers/admin_analytics.py

```
@@ -81,7 +126,7 @@ async def get_overview(request: Request, db=Depends(get_db)):
 # Add missing sessions endpoint
 @router.get("/sessions")
 async def get_sessions(request: Request, db=Depends(get_db)):
-    admin_user = JWTHandler.verify_admin_access(request)
+    admin_user = await AuthenticationHelper.verify_admin_access_strict(request, db)
```

Ruff (0.12.2)

Local variable `admin_user` is assigned to but never used

Remove assignment to unused variable `admin_user`

(F841)